iam.tf

```diff
@@ -0,0 +1,40 @@
+data "aws_iam_policy_document" "log_assume" {
+  statement {
+    actions = ["sts:AssumeRole"]
+
+    principals {
+      type        = "Service"
+      identifiers = ["logs.${length(var.region) > 0 ? var.region: data.aws_region.default.name}.amazonaws.com"]
+    }
+  }
+}
+
+data "aws_iam_policy_document" "log" {
+  statement {
+    actions = [
+      "logs:CreateLogGroup",
+      "logs:CreateLogStream",
+      "logs:PutLogEvents",
+      "logs:DescribeLogGroups",
+      "logs:DescribeLogStreams",
+      "logs:CreateLogStream",
+      "logs:DeleteLogStream",
+    ]
+
+    resources = [
```

Comment:
Use `compact` with `concat`.

Reply:
> Use compact with concat.

It can break the module work - with `json` need be very careful

Reply:
Explain why it can break json

Reply:
I'm sure which delimiter between elements I'll get after the `concat()`

Reply:
It seems, there is no need to use `join` with delimiter here.
https://www.terraform.io/docs/providers/aws/d/iam_policy_document.html